pkg/client/v1/nimbleos/access_control_record.go

```diff
@@ -4,33 +4,51 @@ package nimbleos
 
 // AccessControlRecord - Manage access control records for volumes.
 // Export AccessControlRecordFields for advance operations like search filter etc.
-var AccessControlRecordFields *AccessControlRecord
+var AccessControlRecordFields *AccessControlRecordStringFields
 
 func init() {
 	IDfield := "id"
+	ApplyTofield := "apply_to"
 	ChapUserIdfield := "chap_user_id"
 	ChapUserNamefield := "chap_user_name"
 	InitiatorGroupIdfield := "initiator_group_id"
 	InitiatorGroupNamefield := "initiator_group_name"
+	Lunfield := "lun"
```

Comment:
Is there a reason for these variables to be capitalized?  If it is easy to make them lower case, maybe we should.  I guess it isn't important.  A simple thing would be to put "field" at the beginning, like "fieldLun".

Reply:
Not sure why it needs to be capitalized, I was just following the code in the master branch. Sure, we can change it to `field{UpperCamelCaseOfTheFieldName}`

Reply:
We won't need the field variables at all if we remove the pointers from the struct.

Reply:
Yes, that would be ideal. Would that cause a lot of code churn?